core/src/main/java/org/mule/processor/IdempotentRedeliveryPolicy.java

```diff
@@ -320,6 +325,21 @@ public void setIdExpression(String idExpression)
         this.idExpression = idExpression;
     }
     
+    public ObjectStore<AtomicInteger> getObjectStore()
```

Comment:
Maybe all these objectStore setters should say in the javadoc that the returned value can be different depending if the instance is initialized or not.